python/python/glide/__init__.py

```diff
@@ -106,10 +122,13 @@
 
 __all__ = [
     # Client
+    "BaseClient",
```

Comment:
are these base objects needed for anything?

Reply:
We export it in other clients too

Reply:
We don't want users to create a BaseClient object, and we can prevent that by not exporting it.